### DISCUSSION TITLE
Pore size distribution in compacts

### DISCUSSION THREAD

**Discussion by Mithu1993** [PMEAL/OpenPNM#2364] [Q&A]
Hi,

I have data of the position (x,y,z) and radius of spheres in a compacts (have attached an example file), generated from the DEM.  Can I use your software to calculate the pore size and generate the pore network? If yes, how can I do it.
[PH101_particle_pos_20_1_1e-06.txt](https://github.com/PMEAL/OpenPNM/files/8412164/PH101_particle_pos_20_1_1e-06.txt)

Best regards,
Mithu

**Comment by jgostick** [comment]
I think is best analyzed in porespy, which I see you've also cross-posted to.  I have written an [answer here](https://github.com/PMEAL/porespy/discussions/678)

**Reply by Mithu1993** [reply]
Hi,

Thank you very, sorry I was not sure which of the software I could use for my application.

Best regards,
Mithu
